autrite/pipeline.py

```diff
@@ -41,6 +41,7 @@
         print("[Before filtering DB constraints] ", len(constraints))
         constraints = [c for c in constraints if c.db == True]
         print("[After filtering DB constraints] ", len(constraints))
+    exit(0)
```

Comment:
why exit(0) here?

Reply:
it's a bug, my bad. Now fixed.

Reply:
Why you add from `Provedumper import ProveDumper` here?

Reply:
cuz we use `Provedumper` [here](https://github.com/LiuXiaoxuanPKU/ConstrOpt/blob/da1927b608ecb8b92dd9e46d15eaa99a217c825b/constropt/autrite/pipeline.py#L139).